src/sage/combinat/symmetric_group_algebra.py

```diff
@@ -1941,6 +1941,46 @@ def _dft_seminormal(self, mult='l2r'):
         snb = self.seminormal_basis(mult=mult)
         return matrix([vector(b) for b in snb]).inverse().transpose()
 
+    def _dft_modular(self):
+        r"""
+        Return the discrete Fourier transform when the characteristic divides the order of the group.
+
+        See [Mur1983]_ for the construction of central primitive orthogonal idempotents.
+        For each idempotent `e_i` we have a projection `v \mapsto v*e_i`. This is a homomorphism.
+        We choose a basis for each submodule spanned by `\{\sigma*e_i | \sigma \in S_n\}`.
+        The change-of-basis from the standard basis `\{\sigma\}_\sigma` is returned.
```

Comment:
The docstrings are wrapped to 80 characters (Source: [discussion](https://github.com/sagemath/sage/pull/37118#issuecomment-1960754796))

Reply:
All lines are now under 80 characters (learned this was the IBM punch card limit as well).